components/steps/index.tsx

```diff
@@ -19,7 +19,12 @@ export interface StepsProps {
 }
 
 export default class Steps extends React.Component<StepsProps, any> {
-  static Step = RcSteps.Step;
+  static Step = {
+    description: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+    icon: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+    title: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+    status: PropTypes.string,
+  };
```

Comment:
`RcSteps.Step` is a component, not a propTypes. You need define an typescript def like:
```tsx
static Step = RcSteps.Step as React.ClassicComponentClass<StepProps>;
```

Reply:
ok, let me try!